views/donate.pug

```diff
@@ -11,15 +11,25 @@ html
       let amount = "#{amount}";
       campaignUrl += "?utm_source=" + campaign + "&override_amount=" + amount + "&payment_method=card";
       function redirectToCampaignPage() {
-        setTimeout(function(){ window.location.href = campaignUrl; }, 5000);
+        //- setTimeout(function(){ window.location.href = campaignUrl; }, 5000);
```

Comment:
Did you mean to comment out the time based redirect?

Reply:
No, fixed!